docs/pipelines/tasks/includes/yaml/AzureMysqlDeploymentV1.md

```diff
@@ -12,17 +12,17 @@ ms.technology: devops-cicd-tasks
 # Run your scripts and make changes to your Azure Database for MySQL
 - task: AzureMysqlDeployment@1
   inputs:
-    azureSubscription: 
-    serverName: 
-    #databaseName: # Optional
-    sqlUsername: 
-    sqlPassword: 
-    #taskNameSelector: 'SqlTaskFile' # Optional. Options: sqlTaskFile, inlineSqlTask
-    #sqlFile: # Required when taskNameSelector == SqlTaskFile
-    #sqlInline: # Required when taskNameSelector == InlineSqlTask
-    #sqlAdditionalArguments: # Optional
-    #ipDetectionMethod: 'AutoDetect' # Options: autoDetect, iPAddressRange
-    #startIpAddress: # Required when ipDetectionMethod == IPAddressRange
-    #endIpAddress: # Required when ipDetectionMethod == IPAddressRange
-    #deleteFirewallRule: true # Optional
+    ConnectedServiceName: # Or alias AzureSubscription
```

Comment:
here also it should be azureSubscription, right ? @ninallam

Reply:
Yes it should be azureSubscription

Reply:
Updated :)